homeassistant/components/sensor/wirelesstag.py

```diff
@@ -0,0 +1,168 @@
+"""
+Sensor support for Wirelss Sensor Tags platform.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.wirelesstag/
+"""
+
+import logging
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_ENTITY_NAMESPACE, CONF_MONITORED_CONDITIONS)
+from homeassistant.components.wirelesstag import (
+    DEFAULT_ENTITY_NAMESPACE, DOMAIN as WIRELESSTAG_DOMAIN,
+    WIRELESSTAG_TYPE_13BIT, WIRELESSTAG_TYPE_WATER,
+    WIRELESSTAG_TYPE_ALSPRO,
+    WirelessTagBaseSensor)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import TEMP_CELSIUS
+
+DEPENDENCIES = ['wirelesstag']
+
+_LOGGER = logging.getLogger(__name__)
+
+SENSOR_TEMPERATURE = 'temperature'
+SENSOR_HUMIDITY = 'humidity'
+SENSOR_MOISTURE = 'moisture'
+SENSOR_LIGHT = 'light'
+
+SENSOR_TEMPERATURE_F_ICON = 'temperature-fahrenheit'
```

Comment:
Don't define icons. Instead use device classes for sensors.

Reply:
got it. in progress.

Reply:
icons removed.